cmd/kubeadm/app/cmd/phases/certs.go

```diff
@@ -79,79 +97,62 @@ func NewCmdCerts() *cobra.Command {
 	return cmd
 }
 
-// getCertsSubCommands returns sub commands for certs phase
-func getCertsSubCommands(defaultKubernetesVersion string) []*cobra.Command {
-
-	cfg := &kubeadmapiv1beta1.InitConfiguration{}
-
-	// Default values for the cobra help text
-	kubeadmscheme.Scheme.Default(cfg)
-
-	var cfgPath string
+// getCertsSubPhases returns sub phases for certs phase
+func getCertsSubPhases() []workflow.Phase {
+	subPhases := []workflow.Phase{}
 
-	// Special case commands
-	// All runs CreatePKIAssets, which isn't a particular certificate
-	allCmd := &cobra.Command{
-		Use:     "all",
-		Short:   "Generates all PKI assets necessary to establish the control plane",
-		Long:    allCertsLongDesc,
-		Example: allCertsExample,
-		Run: func(cmd *cobra.Command, args []string) {
-			internalcfg, err := configutil.ConfigFileAndDefaultsToInternalConfig(cfgPath, cfg)
-			kubeadmutil.CheckErr(err)
+	certTree, err := certsphase.GetDefaultCertList().AsMap().CertTree()
+	kubeadmutil.CheckErr(err)
```

Comment:
I find this error handling kind of weird here.
Considering that this error can happen only if there is a misconfiguration of certs, and this is verified by unit test in certlist_test.go, IMO we should simply ignore the error

Reply:
Done